spec/system/composer/post_validation_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+describe "Composer Post Validations", type: :system, js: true do
+  fab!(:tl0_user) { Fabricate(:user, trust_level: 0) }
+  fab!(:tl1_user) { Fabricate(:user, trust_level: 1) }
+  fab!(:tl2_user) { Fabricate(:user, trust_level: 2) }
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:post) { Fabricate(:post, topic: topic) }
+
+  let(:composer) { PageObjects::Components::Composer.new }
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+
+  describe "trust level 0 user" do
```

Comment:
Code ⛳ but we can use `shared_examples` here if the tests for TL0 and TL1 are the same.

Reply:
Cool! Didn't know that existed, just pushed the change now, thanks!